Fixes, text formatting and favicon addition

mh-mazen opened the pull request:
- Update navigator Route in case of meta mask not detected

mh-mazen committed:
Fix Navigation Url in main component

- Update navigator Route in case of meta mask not detected

mh-mazen committed:
Alpha Fix For App Components 

- Fixed Missing declarations in some services
- added testing for individual component
* still missing some fixes for components
- bug fixes
- code cleanup

mh-mazen committed:
Uplload Favicon and Test For Bugs

mh-mazen committed:
Spelling Fix

Commit:
Update Connection Status Messages

Muhammad-Altabba commented on src/app/pages/deployment-network/deployment-network.component.html:
```diff
@@ -46,8 +46,8 @@
 </div>
 <div *ngIf="compareDeployedNetwork()" class="ui-messages ui-messages-success d-block" role="alert">
-  You are on <b>{{ this.web3Service.network }} </b> and are able to use this DApp!
+  You are connected to  '<b>{{ this.web3Service.network }} </b>' ,You can use our DApp!
```
You are connected to  '<b>{{ this.web3Service.network }}</b>'. And you can use this DApp!

Muhammad-Altabba commented on src/app/pages/deployment-network/deployment-network.component.html:
```diff
@@ -46,8 +46,8 @@
-  You are on <b>{{ this.web3Service.network }} </b> and are able to use this DApp!
+  You are connected to  '<b>{{ this.web3Service.network }} </b>' ,You can use our DApp!
 </div>
 <div *ngIf="!compareDeployedNetwork()" class="ui-messages ui-messages-danger d-block" role="alert">
```
Best practices are to use properties instead of methods when hiding/showing html content and similar tasks.

Muhammad-Altabba commented on src/app/pages/deployment-network/deployment-network.component.html:
```diff
@@ -46,8 +46,8 @@
-  You are on <b>{{ this.web3Service.network }} </b> and are not able to use this DApp!
-</div>
\ No newline at end of file
+  You are not able to use this DApp, because you are connected to '<b>{{ this.web3Service.network }} </b>'.
```
If the user does not have an injected Web3 (this.web3Service.network is empty) the message showed to be something like: Use an Ethereum Wallet extension, or an Ethereum DApp-compatible browser, to be able to use this DApp.

Commit:
Fix for Travis-Cli , Routing for deployment

- Fixed Routing for ContractDeploymentNetwork Component
- Travis-cli :
  * Added Messing Imports 
  * Fixed unhanded loaders for services
- Update Contract Deployment address on rinkeby

Commit:
Code Formating, Travis ClI Fix

- Fixed text formatting issues in many files (trailing white space, new line , comma)
- Removed the use of e2e testing as its irrelevant to the app
